pkg/cmd/server/kubernetes/network/network_config.go

```diff
@@ -66,17 +70,21 @@ func New(options configapi.NodeConfig, clusterDomain string, proxyConfig *compon
 
 	var sdnNode network.NodeInterface
 	var sdnProxy network.ProxyInterface
+	var internalNetworkInformers networkinformers.SharedInformerFactory
```

Comment:
You could get rid of these temporary variables if you moved the creation of `config` up here, and then in the "if IsOpenShiftNetworkPlugin" section, you could just set `config.SDNNode`, `config.SDNProxy`, and `config.InternalNetworkInformers`